.github/workflows/nextjs_bundle_analysis.yml

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+name: 'Next.js Bundle Analysis'
+
+on:
+  pull_request:
+  push:
+    branches:
+      - dev
```

Comment:
Do we need this?

Reply:
According to the template this should be the default branch which is dev for us so it can compare bundle sizes with dev later on.